tasks/noci_test.sh

```diff
@@ -24,25 +24,12 @@ test_jasmine () {
     npm run test-jasmine -- --tags=noCI,noCIdep --nowatch || EXIT_STATE=$?
 }
 
-# mapbox image tests take too much resources on CI
-#
-# since the update to mapbox-gl@0.44.0, we must use orca
-# as mapbox-gl versions >0.22.1 aren't supported on nw.js@0.12 used in the
-# 'old' image server
-#
-# due to mathjax upgrade to v2.7.5, we are unable to test mathjax mocks in the
-# 'old' image server and we use orca
-#
+# having problem creating baselines for 2 mapbox mocks using kaleido
+# we must use orca
```

Comment:
But other mapbox mocks work? what happens with these two?

Reply:
I get the following error cc: @jonmmease
```
Transform failed with error code 525: Mapbox error.
```
when try to generate baselines for these two:
1. https://github.com/plotly/plotly.js/blob/master/test/image/mocks/mapbox_density0-legend.json
2. https://github.com/plotly/plotly.js/blob/master/test/image/mocks/mapbox_osm-style.json